File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        char[] arr1=s.toCharArray();
        char[] arr2=t.toCharArray();
        Arrays.sort(arr1);
        Arrays.sort(arr2);
        s=new String(arr1);
        t=new String(arr2);
        if(s.equals(t)){
            return true;
        }
        return false;
    }
}

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        # l1=list(s)
        # l2=list(t)
        # l1.sort()
        # l2.sort()
        # if(l1==l2):
        #     return True
        # return False
        if(len(s)!=len(t)):
            return False
        dic={}
        for i in s:
            dic[i]=dic.get(i,0)+1
        for i in t:
            if i not in dic or dic[i]==0:
                return False
            dic[i]-=1
        return True